lib/_internal/file_picker_web.dart

```diff
@@ -58,7 +60,9 @@ class FilePickerWeb extends FilePicker {
         Completer<List<PlatformFile>?>();
 
     String accept = _fileType(type, allowedExtensions);
-    InputElement uploadInput = FileUploadInputElement() as InputElement;
+    HTMLInputElement uploadInput =
```

Comment:
```suggestion
    HTMLInputElement uploadInput = HTMLInputElement();
```

The constructor calls document.createElement under the hood.

Reply:
refactored